CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 
+- `[expect, jest-snapshot]` Support custom inline snapshot matchers ([#9278](https://github.com/facebook/jest/pull/9278))
```

Comment:
Alphabetical, please 🙂

Reply:
Oops! Sure! (I think this is not mentioned in the CONTRIBUTING file? Or I must have missed it)

Reply:
Sorta new as of this release. A separate pr tweaking the text in the contributing guide would be very welcome as well 👍